File: SNXxj/src/main/java/DBApp.java
import org.SNXex.db.DBS;
import org.SNXex.db.LoadFakeDB;
import org.apache.SNX.SNX;

public class DBApp {

	static DBS _db;

	public static void main(String[] args) throws Throwable {

		new SNX();

		//_db = new DB(5000, ":memory:"); // for memory
		_db = new DBS(5000, "/home/vic/db/sdb.db"); // or file path if not using RAM

		new LoadFakeDB(_db).loadDB(); // load fake data 
	}

	public void other() {
		// new SS();
	}
}

File: SNX_02/src/main/scala/DBApp.scala

import org.SNXex.db.DBS

import org.SNXex.db.LoadFakeDB

import org.apache.SNX.SNX

import DBApp._

object DBApp {

  var _mdb: DBS = _

  def main(args: Array[String]): Unit = {
    new SNX()
    // for memory
    //_mdb = new DBS(5000, ":memory:")
    // or file path if not using RAM
    _mdb = new DBS(5000, "/home/vic/db/sdb.db")
    // load fake data
    new LoadFakeDB(_mdb).loadDB()
  }

}
